bam_sort.c

```diff
@@ -1742,7 +1742,7 @@ static int bam_merge_simple(SamOrder sam_order, char *sort_tag, const char *out,
     heap = (heap1_t*)calloc(heap_size, sizeof(heap1_t));
     if (!heap) goto mem_fail;
 
-    // Make sure that there's enough memory for template coordinate keys
+    // Make sure that there's enough memory for template coordinate keys, one per file to read
```

Comment:
Added this comment, since I kept on getting confused why we're only looking for `n` more keys.  The reason is we're only storing one record per file from the files we're reading in and merging.